hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -64,23 +66,23 @@
     // IPs. See `hw/dv/tools/dvsim/common_sim_cfg.hjson` for the default value.
     {
       name: default_vcs_cov_cfg_file
-      value: "-cm_hier {proj_root}/hw/top_earlgrey/dv/cov/chip_cover.cfg+{proj_root}/hw/top_earlgrey/dv/autogen/xbar_tgl_excl.cfg"
+      value: "-cm_hier {top_dv_path}/cov/chip_cover.cfg+{top_dv_path}/autogen/xbar_tgl_excl.cfg -cm_fsmcfg {top_dv_path}/cov/chip_fsm.cfg"
```

Comment:
Ideally, this would be captured in `vcs_cov` `build_mode` instead as a `build_opts` (since build_opts are additive).

Reply:
this sounds like an addition to cov related config. Which build_mode do you refer to?

Reply:
Right - coverage related settings are added in `vcs_cov` build mode. I was suggesting doing it like this below. 

```suggestion
  build_modes: [
    {
      name: vcs_cov
      build_opts: ["-cm_fsmcfg {top_dv_path}/cov/chip_fsm.cfg"]
    }
  ]
```

The reason for suggesting this change is because `-cm_hier` and `-cm_fsmcfg` are different, unrelated switches.

Reply:
I was thinking to supply this only for the default coverage build, since only chip_cover.cfg enables PLIC coverage, adding it to exclude PLIC-FSM.
I'm fine with this too. Let me see if it works.

Reply:
Yeah ok...I would guess it will throw a warning for the `cover_reg_top` build mode since it only enables coverage on TL ports (and limited other things). I am fine with the existing approach if this does not work. We will hopefully remove it later anyway.

Reply:
yeah, it indeed throws out a warning. Let me switch back to the earlier approach so that we have less warning to clean up.